Found 10 items
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:17 /myfile1
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:17 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:17 /myfile2
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:17 /myfile3
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:17 /myfile4
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:17 /myfile5
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:17 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:17 /myfile7
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:17 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:17 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
2019-02-12 11:18:16,356 INFO retry.RetryInvocationHandler: java.net.ConnectException: Call From clnode031.clemson.cloudlab.us/130.127.133.40 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 1142ms.
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
file 10 has been read from HDFS for 9 time
diff succeed!
file 10 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile10
file 10 has been removed from HDFS
Found 10 items
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:21 /myfile1
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:23 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:21 /myfile2
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:17 /myfile3
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:23 /myfile4
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:21 /myfile5
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:23 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:21 /myfile7
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:21 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:21 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
file 10 has been read from HDFS for 9 time
diff succeed!
file 10 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile10
file 10 has been removed from HDFS
Found 9 items
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:27 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:21 /myfile2
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:25 /myfile3
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:27 /myfile4
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:21 /myfile5
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:27 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:21 /myfile7
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:25 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:25 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
2019-02-12 11:28:43,347 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getFileInfo(FSNamesystem.java:3085)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getFileInfo(NameNodeRpcServer.java:1154)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getFileInfo(ClientNamenodeProtocolServerSideTranslatorPB.java:966)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 1302ms.
2019-02-12 11:28:44,652 INFO retry.RetryInvocationHandler: java.net.ConnectException: Call From clnode031.clemson.cloudlab.us/130.127.133.40 to node-0-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-0-link-0/10.10.1.1:8020 after 2 failover attempts. Trying to failover after sleeping for 2577ms.
file 10 has been read from HDFS for 3 time
diff succeed!
2019-02-12 11:29:19,522 INFO retry.RetryInvocationHandler: java.net.ConnectException: Call From clnode031.clemson.cloudlab.us/130.127.133.40 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 597ms.
2019-02-12 11:29:20,122 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getFileInfo(FSNamesystem.java:3085)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getFileInfo(NameNodeRpcServer.java:1154)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getFileInfo(ClientNamenodeProtocolServerSideTranslatorPB.java:966)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-0-link-0/10.10.1.1:8020 after 2 failover attempts. Trying to failover after sleeping for 1765ms.
2019-02-12 11:29:21,890 INFO retry.RetryInvocationHandler: java.net.ConnectException: Call From clnode031.clemson.cloudlab.us/130.127.133.40 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 3 failover attempts. Trying to failover after sleeping for 4804ms.
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
file 10 has been read from HDFS for 9 time
diff succeed!
file 10 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile10
file 10 has been removed from HDFS
Found 9 items
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:32 /myfile1
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:32 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:32 /myfile3
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:31 /myfile4
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:21 /myfile5
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:27 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:29 /myfile7
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:31 /myfile8
-rw-r--r--   3 root supergroup 2013265920 2019-02-12 11:32 /myfile9._COPYING_
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
file 10 has been read from HDFS for 9 time
diff succeed!
file 10 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile10
file 10 has been removed from HDFS
Found 9 items
-rw-r--r--   3 root supergroup 2013265920 2019-02-12 11:37 /myfile1._COPYING_
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:37 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:33 /myfile2
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:32 /myfile3
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:34 /myfile5
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:36 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:34 /myfile7
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:36 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:32 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
2019-02-12 11:40:26,087 INFO retry.RetryInvocationHandler: java.io.EOFException: End of File Exception between local host is: "clnode031.clemson.cloudlab.us/130.127.133.40"; destination host is: "node-1-link-0":8020; : java.io.EOFException; For more details see:  http://wiki.apache.org/hadoop/EOFException, while invoking ClientNamenodeProtocolTranslatorPB.getBlockLocations over node-1-link-0/10.10.1.4:8020. Trying to failover immediately.
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
file 10 has been read from HDFS for 9 time
diff succeed!
file 10 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile10
file 10 has been removed from HDFS
Found 10 items
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:42 /myfile1
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:42 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:41 /myfile2
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:38 /myfile3
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:38 /myfile4
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:38 /myfile5
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:41 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:38 /myfile7
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:41 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:40 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
file 10 has been read from HDFS for 9 time
diff succeed!
file 10 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile10
file 10 has been removed from HDFS
Found 10 items
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:47 /myfile1
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:48 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:45 /myfile2
-rw-r--r--   3 root supergroup 2013265920 2019-02-12 11:48 /myfile3._COPYING_
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:43 /myfile4
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:48 /myfile5
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:47 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:48 /myfile7
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:47 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:45 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
2019-02-12 11:51:29,974 INFO retry.RetryInvocationHandler: java.io.EOFException: End of File Exception between local host is: "clnode031.clemson.cloudlab.us/130.127.133.40"; destination host is: "node-1-link-0":8020; : java.io.EOFException; For more details see:  http://wiki.apache.org/hadoop/EOFException, while invoking ClientNamenodeProtocolTranslatorPB.getBlockLocations over node-1-link-0/10.10.1.4:8020. Trying to failover immediately.
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
file 10 has been read from HDFS for 9 time
diff succeed!
file 10 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile10
file 10 has been removed from HDFS
Found 10 items
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:52 /myfile1
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:53 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:50 /myfile2
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:48 /myfile3
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:50 /myfile4
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:48 /myfile5
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:52 /myfile6
-rw-r--r--   3 root supergroup  671088640 2019-02-12 11:53 /myfile7._COPYING_
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:52 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:50 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
file 10 has been read from HDFS for 9 time
diff succeed!
file 10 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile10
file 10 has been removed from HDFS
Found 8 items
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:57 /myfile1
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:58 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:54 /myfile2
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:55 /myfile4
-rw-r--r--   3 root supergroup          0 2019-02-12 11:58 /myfile5._COPYING_
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:56 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:57 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:56 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
file 10 has been read from HDFS for 9 time
diff succeed!
2019-02-12 12:02:33,360 INFO retry.RetryInvocationHandler: java.io.EOFException: End of File Exception between local host is: "clnode031.clemson.cloudlab.us/130.127.133.40"; destination host is: "node-1-link-0":8020; : java.io.EOFException; For more details see:  http://wiki.apache.org/hadoop/EOFException, while invoking ClientNamenodeProtocolTranslatorPB.getBlockLocations over node-1-link-0/10.10.1.4:8020. Trying to failover immediately.
file 10 has been read from HDFS for 10 time
diff succeed!
2019-02-12 12:02:56,314 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile10. Name node is in safe mode.
The reported blocks 144 has reached the threshold 0.9990 of total blocks 144. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 7 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile10. Name node is in safe mode.
The reported blocks 144 has reached the threshold 0.9990 of total blocks 144. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 7 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-0-link-0/10.10.1.1:8020. Retrying immediately.
2019-02-12 12:02:56,317 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile10. Name node is in safe mode.
The reported blocks 144 has reached the threshold 0.9990 of total blocks 144. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 7 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile10. Name node is in safe mode.
The reported blocks 144 has reached the threshold 0.9990 of total blocks 144. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 7 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-0-link-0/10.10.1.1:8020. Retrying after sleeping for 919ms.
2019-02-12 12:02:57,241 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile10. Name node is in safe mode.
The reported blocks 144 has reached the threshold 0.9990 of total blocks 144. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 6 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile10. Name node is in safe mode.
The reported blocks 144 has reached the threshold 0.9990 of total blocks 144. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 6 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-0-link-0/10.10.1.1:8020. Retrying after sleeping for 1853ms.
2019-02-12 12:02:59,097 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile10. Name node is in safe mode.
The reported blocks 144 has reached the threshold 0.9990 of total blocks 144. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 4 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile10. Name node is in safe mode.
The reported blocks 144 has reached the threshold 0.9990 of total blocks 144. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 4 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-0-link-0/10.10.1.1:8020. Retrying after sleeping for 3045ms.
2019-02-12 12:03:02,145 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile10. Name node is in safe mode.
The reported blocks 144 has reached the threshold 0.9990 of total blocks 144. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 1 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile10. Name node is in safe mode.
The reported blocks 144 has reached the threshold 0.9990 of total blocks 144. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 1 seconds. NamenodeHostName:node-0-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-0-link-0/10.10.1.1:8020. Retrying after sleeping for 5409ms.
Deleted /myfile10
file 10 has been removed from HDFS
Found 10 items
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:03 /myfile1
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:03 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:00 /myfile2
-rw-r--r--   3 root supergroup 1610612736 2019-02-12 12:03 /myfile3._COPYING_
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:00 /myfile4
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:58 /myfile5
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:02 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:03 /myfile7
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 11:57 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:00 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
file 10 has been read from HDFS for 9 time
diff succeed!
file 10 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile10
file 10 has been removed from HDFS
Found 10 items
-rw-r--r--   3 root supergroup 1476395008 2019-02-12 12:08 /myfile1._COPYING_
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:08 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:05 /myfile2
-rw-r--r--   3 root supergroup  134217728 2019-02-12 12:08 /myfile3._COPYING_
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:05 /myfile4
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:05 /myfile5
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:07 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:08 /myfile7
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:05 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:07 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
2019-02-12 12:13:37,416 INFO retry.RetryInvocationHandler: java.io.EOFException: End of File Exception between local host is: "clnode031.clemson.cloudlab.us/130.127.133.40"; destination host is: "node-1-link-0":8020; : java.io.EOFException; For more details see:  http://wiki.apache.org/hadoop/EOFException, while invoking ClientNamenodeProtocolTranslatorPB.getBlockLocations over node-1-link-0/10.10.1.4:8020. Trying to failover immediately.
file 10 has been read from HDFS for 9 time
diff succeed!
file 10 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile10
file 10 has been removed from HDFS
Found 10 items
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:13 /myfile1
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:14 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:10 /myfile2
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:13 /myfile3
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:10 /myfile4
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:10 /myfile5
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:12 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:13 /myfile7
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:10 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:12 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
file 10 has been read from HDFS for 7 time
diff succeed!
file 10 has been read from HDFS for 8 time
diff succeed!
file 10 has been read from HDFS for 9 time
diff succeed!
file 10 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile10
file 10 has been removed from HDFS
Found 10 items
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:18 /myfile1
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:20 /myfile10
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:16 /myfile2
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:18 /myfile3
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:16 /myfile4
-rw-r--r--   3 root supergroup  268435456 2019-02-12 12:20 /myfile5._COPYING_
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:18 /myfile6
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:18 /myfile7
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:16 /myfile8
-rw-r--r--   3 root supergroup 2147483648 2019-02-12 12:18 /myfile9
file 10 has been put into HDFS
file 10 has been read from HDFS for 1 time
diff succeed!
file 10 has been read from HDFS for 2 time
diff succeed!
file 10 has been read from HDFS for 3 time
diff succeed!
file 10 has been read from HDFS for 4 time
diff succeed!
file 10 has been read from HDFS for 5 time
diff succeed!
file 10 has been read from HDFS for 6 time
diff succeed!
signal TERM catched and let quit gracefully
Deleted /myfile10
file 10 has been removed from HDFS
